ansible/roles/kind/kind_cluster_create/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Render kind config
+  block:
+    - template:
+        src: kind.yaml.j2
+        dest: /tmp/kind.yaml
```

Comment:
I agree, It will be great to have kind config as a build artifact. 
Also, there is a `tempfile` module for ansible [1] which can be used for tmp files generation.
[1] https://docs.ansible.com/ansible/2.9/modules/tempfile_module.html

Reply:
Added